cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -162,6 +163,7 @@ func NewUpdateCmd(o *CreateOrUpdateOptions, flagsFactory cmdcore.FlagsFactory) *
 	cmd.Flags().StringVarP(&o.packageName, "package", "p", "", "Name of package install to be updated")
 	cmd.Flags().StringVar(&o.version, "version", "", "Set package version")
 	cmd.Flags().StringVar(&o.valuesFile, "values-file", "", "The path to the configuration values file, optional")
+	cmd.Flags().BoolVar(&o.dropValuesFile, "drop-values-file", false, "Drop values file and remove reference from installation if values are not supplied, optional")
```

Comment:
it seems that drop-values-file is too specific of a name for the feature since it's removing all values. i wonder if we should really do `--values=bool` (with default=true) or something along those lines that represents that no values will be set on the packageinstall.

Reply:
Just `--values` makes sense to me as well.
In that case the behaviour would be that the provided values are ignored, if the flag is set to false imo.

`--with-values` might be more instinctive

Reply:
> provided values are ignored, if the flag is set to false imo.

may be even raise an error?

Reply:
Sounds good!

Reply:
Went ahead with `--values` as it might make more sense since it is in create, update and install commands